meteor/client/ui/Shelf/Keyboard/KeyboardPreview.tsx

```diff
@@ -27,22 +27,6 @@ import { translateMessage } from '@sofie-automation/corelib/dist/TranslatableMes
 
 const _isMacLike = navigator.platform.match(/(Mac|iPhone|iPod|iPad)/i) ? true : false
 
-declare global {
```

Comment:
You don't want the files in metoer/ui/Shelf/Keyboard
In the gitignore there should be some comments that need to be uncommented so you don't get these file.